examples/fake-ad.amp.html

```diff
@@ -0,0 +1,24 @@
+<!doctype html>
```

Comment:
My preference would be to leave it in examples/ so that an AMP page author that wants to write a page and test including ads in it can use this. But it's up to you. Do you think this is valid/useful?